docs/manifest-types.md

```diff
@@ -357,13 +357,78 @@ This type of manifest adds a navigation entry into the navigation.xml.
                 "app-catalog-gui": {
                     "label": "Apps",
                     "title": "Apps",
+                    "bundle": "app-catalog-gui",
                     "icon": "fa-archive",
                     "module": "app-catalog-gui",
                     "controller": "index",
                     "action": "index"
                 }
             },
             "after": "users"
+        },
+        {
+            "navigations": {
+                "main": {
+                    "pages": {
+                        "main-nested": {
+                            "bundle": "main",
+                            "controller": "index",
+                            "action": "index",
+                            "visible": "1"
+                        }
+                    }
+                }
+            },
+            "before": "before-main"
```

Comment:
I understood that new examples shows how to add new page to the existing "main" navigation element. If it is right, then we don't need this line here:

```
  "before": "before-main"
```

Reply:
Deleted ++